code/_onclick/hud/ai.dm

```diff
@@ -1,3 +1,25 @@
+/obj/screen/ai/multicam 
+	name = "Multicamera Mode"
+	icon = 'icons/mob/screen_ai.dmi'
+	icon_state = "multicam"
+ 
+/obj/screen/ai/multicam/Click() 
+    if(..()) 
+        return 
+    var/mob/living/silicon/ai/AI = usr 
+    AI.toggle_multicam() 
```

Comment:
Are there verbs that can be used interchangeably with these UI buttons?

Reply:
Uh, as far as I'm aware there are no verbs, it's just buttons? I can check, but I don't think there's verbs for it, same with the rest of the AI UI bits.

Reply:
There was a great deal of trouble when kelenius first introduced the ui buttons and simultaneously removed the verbs corresponding to those buttons, and then vehemently refused to add the verbs back. As far as I can tell, making these functions verbs really just involves either setting the procs these buttons are calling as verbs, or otherwise making a verb that calls the same procs in the same manner.

In the interest of maintaining uniform usability, where UI buttons augment what's already available via verbs, I would like verbs that achieve the same effect as the buttons. Some people use trackpads, or are otherwise heavily inconvenienced by trying to use buttons where being able to use their keyboard would be quicker and simpler.

Reply:
I don't really know how to make verbs for it, I'm sorry.